### ISSUE TITLE
Data Preparation for TUMTraf Intersection Dataset does not generates any train and val samples in tumtraf_i_processed directory.

### ISSUE THREAD

**Issue by Uchan1996** [tum-traffic-dataset/coopdet3d#2]
@walzimmer 
Hi, thank you for your great work.
I want to do data preparation code for Intersection dataset.
I succeeded in splitting the intersection dataset into train and val as README in tum-traffic-dataset-dev-kit repository says.
However, data preparation by `create_tumtraf_data.py` in this repository does not generates any train and val samples in tumtraf_i_processed directory.
How can I fix these things?
The output of `create_tumtraf_data.py` is displayed as below:

`split:  ['training', 'validation']
Start converting ...
Converting split: training...
train sample: 0
Converting split: validation...
val sample: 0

Finished ...
creating groundtruth database
Create GT Database of TUMTrafNuscDataset
DATASETS:  Registry(name=dataset, items={'CustomDataset': <class 'mmdet.datasets.custom.CustomDataset'>, 'CocoDataset': <class 'mmdet.datasets.coco.CocoDataset'>, 'CityscapesDataset': <class 'mmdet.datasets.cityscapes.CityscapesDataset'>, 'CocoPanopticDataset': <class 'mmdet.datasets.coco_panoptic.CocoPanopticDataset'>, 'ConcatDataset': <class 'mmdet.datasets.dataset_wrappers.ConcatDataset'>, 'RepeatDataset': <class 'mmdet.datasets.dataset_wrappers.RepeatDataset'>, 'ClassBalancedDataset': <class 'mmdet.datasets.dataset_wrappers.ClassBalancedDataset'>, 'MultiImageMixDataset': <class 'mmdet.datasets.dataset_wrappers.MultiImageMixDataset'>, 'DeepFashionDataset': <class 'mmdet.datasets.deepfashion.DeepFashionDataset'>, 'LVISV05Dataset': <class 'mmdet.datasets.lvis.LVISV05Dataset'>, 'LVISDataset': <class 'mmdet.datasets.lvis.LVISV05Dataset'>, 'LVISV1Dataset': <class 'mmdet.datasets.lvis.LVISV1Dataset'>, 'XMLDataset': <class 'mmdet.datasets.xml_style.XMLDataset'>, 'VOCDataset': <class 'mmdet.datasets.voc.VOCDataset'>, 'WIDERFaceDataset': <class 'mmdet.datasets.wider_face.WIDERFaceDataset'>, 'Custom3DDataset': <class 'mmdet3d.datasets.custom_3d.Custom3DDataset'>, 'NuScenesDataset': <class 'mmdet3d.datasets.nuscenes_dataset.NuScenesDataset'>, 'TUMTrafNuscDataset': <class 'mmdet3d.datasets.tumtraf_dataset.TUMTrafNuscDataset'>, 'TUMTrafV2XNuscDataset': <class 'mmdet3d.datasets.tumtraf_v2x_dataset.TUMTrafV2XNuscDataset'>, 'CBGSDataset': <class 'mmdet3d.datasets.dataset_wrappers.CBGSDataset'>})
completed: 0, elapsed: 0s
`

**Comment by Uchan1996** [comment]
@walzimmer 
I think the paths in `tools/data_converter/tumtraf_converter.py` are wrong which are set from the line 97 to 104.
tumtraf_i directory includes train and val directories.
Each of the two directories has three directories (images, labels_point_clouds, point_clouds).
However, the line from 97 to 104 try to access to `labels` directory which is not included in train and val directories.

**Comment by lacie-life** [comment]
@Uchan1996 are you training TUMTraf-i dataset successfully?

**Comment by Uchan1996** [comment]
@lacie-life 
No. I've gave up using TUMTraf-I dataset.
I was able to use TUMTraf-V2X dataset for training and evaluation.
However, I found the following problems on TUMTraf-V2X dataset.

1. There are many bugs in TUMTraf-V2X code. You need to fix them by yourself.

2. The dataset size is very small as follows:
train: 800 frames
val: 100 frames
test: 100 frames

3. The data split of train, val and test are composed of the frames sampled from the same sequences.
I think this data split is not suitable for evaluation of trained models. The trained models just overfit the specific sequences.
